### ISSUE TITLE
[Question] Why does fixed weight decay support only "one cylce" lr scheduler?

### ISSUE THREAD

**Issue by s-ryosky** [poodarchu/Det3D#21]
In SECOND's repo adam optimizer with fixed weight decay is supported on all lr scheduler.
However, in this repo fixed weight decay is supported on only "one cycle" lr scheduler.
Why?
```
def build_one_cycle_optimizer(model, optimizer_config):
    if optimizer_config.fixed_wd:
        optimizer_func = partial(
            torch.optim.Adam, betas=(0.9, 0.99), amsgrad=optimizer_config.amsgrad
        )
    else:
        optimizer_func = partial(torch.optim.Adam, amsgrad=optimizer_cfg.amsgrad)

    optimizer = OptimWrapper.create(
        optimizer_func,
        3e-3,
        get_layer_groups(model),
        wd=optimizer_config.wd,
        true_wd=optimizer_config.fixed_wd,
        bn_wd=True,
    )

    return optimizer
```

**Comment by poodarchu** [comment]
you can implement it and make a merge request, thanks.
